fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/CreateMTMVInfo.java

```diff
@@ -88,7 +88,7 @@
  */
 public class CreateMTMVInfo {
     public static final Logger LOG = LogManager.getLogger(CreateMTMVInfo.class);
-    public static final String MTMV_PLANER_DISABLE_RULES = "OLAP_SCAN_PARTITION_PRUNE";
+    public static final String MTMV_PLANER_DISABLE_RULES = "OLAP_SCAN_PARTITION_PRUNE,PRUNE_EMPTY_PARTITION";
```

Comment:
@seawinde disable this rule will lead to mv have more partition than query?

Reply:
the mv plan used by query rewrite is generated by `org.apache.doris.mtmv.MTMVCache#from` which runs the same rules as query.
`MTMV_PLANER_DISABLE_RULES` is only used by mv meta check when create mtmv. it seems has no problems